### DISCUSSION TITLE
Is it possible to use a database with only one table for vertices and one table for edges with Sqlg?

### DISCUSSION THREAD

**Discussion by kristianhelgesen** [pietermartin/sqlg#505] [Q&A] [answered]
I'm exploring solutions to expose an existing Postgres database through Tinkerpop, and Sqlg seems to be a really good alternative. 

The only caveat is that my database schema has one table for vertices and one for edges, with a column for the type/label. 
Would it be possible to use the the Sqlg Topology to specify this? Or would it require changes to the implementation?

**Comment by pietermartin** [comment, marked as answer]
No afraid that won't be possible. The best I can suggest to write a program to port it into Sqlg.

**Reply by kristianhelgesen** [reply]
Ok, thank you for the quick reply!
